kubernetes-maven-plugin/doc/src/main/asciidoc/inc/_introduction.adoc

```diff
@@ -165,8 +164,7 @@ mvn package {goal-prefix}:build
 ----
 
 This will create the Docker image against a running Docker daemon (which must be accessible either via Unix Socket or
-with the URL set in `DOCKER_HOST`). Alternatively, when connected to an OpenShift cluster (or using the `openshift`
-<<build-mode,mode>> explicitly), then a Docker build will be performed on OpenShift which at the end creates an
+with the URL set in `DOCKER_HOST`). Alternatively, when connected to an OpenShift cluster then a Docker build will be performed on OpenShift which at the end creates an
```

Comment:
Unless configured, default builds for OpenShift are S2I, I guess we should reflect this;
```suggestion
with the URL set in `DOCKER_HOST`). Alternatively, when connected to an OpenShift cluster then an S2I build will be performed on OpenShift which at the end creates an
```